deploy/kube/localregistry.yaml

```diff
@@ -0,0 +1,88 @@
+# after creating the local registry, to push images to it you need to open up a port to the pod's registry:
```

Comment:
This looks like a private docker registry. Is this really needed for the demo ?
I think it detracts from the rest of the flow.